src/app/screens/LNURLAuth/index.tsx

```diff
@@ -16,17 +16,17 @@ import utils from "~/common/lib/utils";
 import type { LNURLAuthServiceResponse } from "~/types";
 
 function LNURLAuth() {
-  const navigate = useNavigate();
+  const { t } = useTranslation("translation", { keyPrefix: "lnurlauth" });
+  const { t: tComponents } = useTranslation("components", {
+    keyPrefix: "confirm_or_cancel",
+  });
 
+  const navigate = useNavigate();
   const navState = useNavigationState();
 
   const details = navState.args?.lnurlDetails as LNURLAuthServiceResponse;
   const origin = navState.origin;
 
-  const { t } = useTranslation("components", {
-    keyPrefix: "confirm_or_cancel",
-  });
-
   const [successMessage, setSuccessMessage] = useState("");
   const [errorMessage, setErrorMessage] = useState("");
```

Comment:
Maybe we can translate the error message: "Error: Auth status is not ok" too as we show it inside the component itself instead of a toast